README.md

```diff
@@ -1,87 +1,90 @@
 # VisCy
 
-VisCy is a deep learning pipeline for training and deploying computer vision models for image-based phenotyping at single-cell resolution.
-
-The following methods are being developed:
+VisCy (abbreviation of `vision` and `cyto`) is a deep learning pipeline for training and deploying computer vision models for image-based phenotyping at single-cell resolution.
 
+This repository provides a pipeline for the following.
 - Image translation
   - Robust virtual staining of landmark organelles
 - Image classification
   - Supervised learning of of cell state (e.g. state of infection)
 - Image representation learning
   - Self-supervised learning of the cell state and organelle phenotypes
 
-<div style="border: 2px solid orange; padding: 10px; border-radius: 5px; background-color: #fff8e1;">
+<div style="border: 2px solid orange; padding: 10px; border-radius: 5px">
   <strong>Note:</strong><br>
-  VisCy is currently considered alpha software and is under active development. Frequent breaking changes are expected.
+  VisCy has been extensively tested for image translation task. The code for other tasks is under active development. Frequent breaking changes are expected in the main branch as we unify the codebase for above tasks. If you are looking for a well-tested version for virtual staining, please use release 0.2.1 from PyPI.
 </div>
 
 ## Virtual staining
-### Pipeline
-A full illustration of the virtual staining pipeline can be found [here](docs/virtual_staining.md).
-
-### Library of virtual staining (VS) models
-The robust virtual staining models (i.e *VSCyto2D*, *VSCyto3D*, *VSNeuromast*), and fine-tuned models can be found [here](https://github.com/mehta-lab/VisCy/wiki/Library-of-virtual-staining-(VS)-Models)
 
 ### Demos 
-#### Image-to-Image translation using VisCy
-- [Guide for Virtual Staining Models](https://github.com/mehta-lab/VisCy/wiki/virtual-staining-instructions):
-Instructions for how to train and run inference on ViSCy's virtual staining models (*VSCyto3D*, *VSCyto2D* and *VSNeuromast*) 
+- [Virtual staining exercise](./examples/virtual_staining/dlmbl_exercise/solution.ipynb):
```

Comment:
Will this link work on PyPI?

Reply:
Just in case, I will make these absolute paths